components/SplitDisplay.vue

```diff
@@ -228,6 +247,86 @@ export default class SplitDisplay extends Vue {
     GlobalEventEmitter.$emit('setCurrentSplit', this.split);
   }
 
+  mergePreviousSplit(){
+    whithLoad(() => {
+      const chosenSplitTimes = asArray(this.split.SegmentHistory.Time);
+      const previousSplitTimes = asArray(this.segments[this.splitIndex - 1].SegmentHistory.Time);
+
+      let mergedGameTime = secondsToLivesplitFormat(0);
```

Comment:
I don't think you need these two variables, below you can just do 
```TS
previousSplitTimes[chosenSplitIndex].RealTime = secondsToLivesplitFormat(sumRT);
```
without the intermediate variable